assignments/prototype-refactor.js

```diff
@@ -7,3 +7,149 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
```

Comment:
Great use of your initial commit. This is a major and significant change in the file and you've done great committing it. Great code and great content!